runtime-sdk/src/modules/consensus_accounts/mod.rs

```diff
@@ -270,13 +294,15 @@ impl<Accounts: modules::accounts::API, Consensus: modules::consensus::API>
         Ok(types::AccountBalance { balance })
     }
 
+    #[handler(query = "consensus.Account")]
     fn query_consensus_account<C: Context>(
         ctx: &mut C,
         args: types::ConsensusAccountQuery,
     ) -> Result<ConsensusAccount, Error> {
         Consensus::account(ctx, args.address).map_err(|_| Error::InvalidArgument)
     }
 
+    #[handler(message_result = "consensus.TransferFromRuntime")]
```

Comment:
Would it be possible to use a constant here (e.g. `CONSENSUS_TRANSFER_HANDLER`)? Because otherwise this could get out of sync with the emitted message in `withdraw` above. Similar for the below one.